docs/user-guide/usage/cli.md

```diff
@@ -16,14 +16,58 @@ In addition to the [standard options](options.md), the CLI accepts:
 
 The process exits without throwing an error when glob pattern matches no files.
 
+### `--cache-location`
+
+Path to a file or directory for the cache location. More info about this option in [standard options](options.md#cacheLocation).
+
+### `--cache`
+
+Store the results of processed files so that stylelint only operates on the changed ones. By default, the cache is stored in `./.stylelintcache` in `process.cwd()`. More info about this option in [standard options](options.md#cache).
```

Comment:
Wording of the following could be better:

> More info about this option in [standard options](options.md#cache).

Any suggestions?

Link is necessary, because not everything from the option description is copied to this page.